### ISSUE TITLE
Option to upload a video?

### ISSUE THREAD

**Issue by RSKothari** [colinquirk/ShinyEyes#7]
It would be fantastic if you could provide an option to upload a video.
The video automatically moves to the start of the time position marked within the GUI. That way, folks could playback segments of interest as they are labeling. It would also be great if there was a little time mark which went forward with the video to show accurate frame positioning.

Great tool Colin. Perhaps our own tool could provide some insight/ideas for you? https://www.nature.com/articles/s41598-020-59251-5

**Comment by colinquirk** [comment]
I am definitely interested in adding options like this and #1 in order to help provide additional context when labeling data, so I will definitely keep this in mind. It will probably require an entire rethinking of how to deal with these sorts of things in general.

Thank you for sharing the paper, I hope that ShinyEyes can potentially make it easier to generate training sets for machine learning classification tools (and I would love to one day add them into the tool itself so that instead of providing labels it can generate them for you).
